### PR TITLE
Popup menu

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
In this project I added linters for javascript .I also used the following languages
-HTML for markup
-CSS for styling
and
-Javascript for functionality where i made the mobile menu to pop up and when clicked taking users to different sections of the page 
Below is a picture where i ran my liners locally in the vs code it shows i have no errors but in my github it shows i have error
Screenshot (79)
When I run my linters in github it gives this 
Screenshot (80)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -11,6 +11,58 @@ ul li {
...
+  display: flex;
+  flex-direction: column;
+  background-color: #6070ff;
+  height: 100vh;
+  width: 100%;
+  position: absolute;
+  opacity: 0.8;
+}
+
+nav img {
```
Comment (Kossi-stack) [inline style.css:24]: - [ ] When I clicked on the navbar the mobile menu does not disappear, kindly fix it

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @Ngufor-emmanuella,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).



 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -11,20 +11,31 @@
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
+<body>
+
```
Comment (tufoinnkuo10) [inline index.html:15]: - [optional] Great job implementing the previous reviewer's changes. kindly make the mobile popup menu not function when you switch to the desktop version. 
Screenshot (174)

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Ngufor-emmanuella   

Your project is complete! Great job for adding the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request ✔️ 
- Validations correctly implemented✔️ 
- menu popsup as expected✔️ 
- menu links working well✔️ 
- No linter errors✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**